extensions/shared/upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { Button } from '@wordpress/components';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import getSiteFragment from '../get-site-fragment';
+
+import './style.scss';
+
+export default ( { feature, plan } ) => (
+	<div className="upgrade-nudge">
```

Comment:
cc @drw158 in case there's a common pattern that could eventually be extracted form this for `wordpress/components` (call to action? banner action?)

Reply:
Just "Card" seems like a good primitive to have in the core?

Reply:
Probably, yes.